src/createCli.js

```diff
@@ -1,16 +1,22 @@
 import yargs from 'yargs';
-import prompt from './prompt';
-import { DEFAULT_STARTER } from './config';
-import createPackageFile from './createPackageFile';
+import prompt from './prompt.js';
+import {
+  DEFAULT_STARTER,
+  FRAMEWORK_CHOICES,
+  STARTER_TYPE_CHOICES,
+} from './config.js';
+import createPackageFile from './createPackageFile.js';
 
-const promisify = (fn) => (...args) => {
-  Promise.resolve(fn(...args)).then(
-    () => {
-      process.exit(0);
-    }
-    // err => report.panic(err)
-  );
-};
+const promisify =
+  (fn) =>
+  (...args) => {
+    Promise.resolve(fn(...args)).then(
+      () => {
+        process.exit(0);
+      }
+      // err => report.panic(err)
```

Comment:
FYI, I think I left the reference to `report` here because it was a good option for a logger.